web/src/tools/testing/bulk_rendering/renderer_core.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable prefer-const */
```

Comment:
Is this still needed? I'd prefer not to be adding lint disablers if we can avoid it

Reply:
Ah, sorry, didn't notice that that sneaked in. Caused by a mis-click. I had wondered why there were still `let` instead of `const` after I went through the file. Now I know why :smile: